### DISCUSSION TITLE
Disable drag selection

### DISCUSSION THREAD

**Discussion by ZennerBlade** [nhn/tui.calendar#1323] [Q&A]
Hi,

Is there a way to disable drag selection on calendar.
For calendar type month, our use-case is to just click an empty date and add an event. I can't seem to find a way that only enables click and disables dragging for selection.
Is there a way to achieve this?

**Comment by adhrinae** [comment]
https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/options.md#gridselection

This option can do the job.

**Reply by ZennerBlade** [reply]
Grid selection doesn't disable the dragging on calendar.
If you set the gridSelection to false, it just disables any kind of action on grid, and if you set enableClick or enableDblClick to false, it disables clicking and double clicking on a particular cell but doesn't stop you from dragging.

I just want the drag to be disabled, is there a way to do just that?

**Reply by adhrinae** [reply]
Somehow I misunderstood your question because selecting date and time also needs dragging.
If you want to prevent dragging each events, you'll need to set the `isReadOnly` property to them.

https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/event-object.md#isreadonly

Similar to #869, #921.

**Reply by ZennerBlade** [reply]
`isReadonly` disables all click actions along with drag.

I want the click and double click to work.
Just want to disable drag.
How should I achieve that?

**Reply by adhrinae** [reply]
Then I'm afraid to say that there's no option to disable dragging only.

**Reply by ZennerBlade** [reply]
Well, I guess we'll have to raise a feature request.
We already have enableClick and enableDblClick as part of gridSelection, might as well add enableDrag

**Reply by adhrinae** [reply]
Wait, Have you read the linked issues I've mentioned?
You can assign the `isReadOnly` to each event to prevent dragging only.
Even though it's just a workaround, This can be enough for you. Check out the example below.

https://stackblitz.com/edit/js-gftomu?file=index.js
